doc/nrf/links.txt

```diff
@@ -52,6 +54,8 @@
 
 .. _`known issues for nRF Connect SDK v1.4.0`: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/known_issues.html?v=v1-4-0
 
+.. _`known issues for nRF Connect SDK v1.4.1`: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/known_issues.html?v=v1-4-1
```

Comment:
Make sure to add that version on master, otherwise the filtering won't work.

Reply:
Yes, planned in this PR: https://github.com/nrfconnect/sdk-nrf/pull/3382